cmake/modules/extensions.cmake

```diff
@@ -1617,6 +1617,30 @@ function(zephyr_build_string outvar)
   set(${outvar} ${${outvar}} PARENT_SCOPE)
 endfunction()
 
+# Function to add directory/es to the include list to be passed to syscall generator.
```

Comment:
`/es`? If plural would be ies. Would just reword to have e.g. `Function to add one or more directories`

Reply:
thanks, reworded